bindings/CXX11/adios2/cxx11/Variable.h

```diff
@@ -348,6 +348,8 @@ class Variable
         size_t Step = 0;
         /** reference to internal block data (used by inline Engine) */
         const T *Data() const;
+        /** true: Dims were swapped from column-major, false: not swapped */
```

Comment:
minor comment: dims may also have been swapped from row-major to col-major, when reading from Fortran.

Reply:
@germasch this particular struct is only exposed in the C++11 bindings